.github/scripts/visualize_stopping_patterns.py

```
@@ -0,0 +1,439 @@
+    return result.stdout
+
+
+def parse_csv_line(header: list[str], line: str) -> dict[str, str] | None:
+    """Parse a single CSV line using csv.reader for proper quoting."""
+    if line.startswith("\\ No newline at end of file"):
+        return None
+    reader = csv.reader(io.StringIO(line))
+    for row in reader:
+        if len(row) < len(header):
+            return None
+        return dict(zip(header, row))
```

Ruff (0.15.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)